src/shared/seccomp-util.c

```diff
@@ -197,40 +227,26 @@ const SystemCallFilterSet syscall_filter_sets[] = {
                 "_sysctl\0"
                 "afs_syscall\0"
                 "break\0"
-                "fattach\0"
-                "fdetach\0"
+                "create_module\0"
                 "ftime\0"
                 "get_kernel_syms\0"
-                "get_mempolicy\0"
```

Comment:
vm86 is actually moved to `@cpu-emulation` and that's documented explicitly in the commit msg.

`get_mempolicy()` isn't obsolete, it's a NUMA thing, hence I removed it, but yeah this should have been mentioned in the commit msg.

So regarding the "not implemented" thing: I figured the syscalls that are not implemented but actually got a syscall nr assigned should be included in `@obsolete`, and the syscalls that ere neither implemented nor a syscall nr assigned should not.  `getpmsg()` is one of the latter, `getmsg()` is of the former, hence I removed `getmsg()` but left `getpmsg()` in there...

The documentation of `@obsolete` says explicitly that it masks non-implemented syscalls, hence I think it should really do that.